src/Domain/Structure/ElementId.hpp

```diff
@@ -49,6 +49,18 @@ class er;
 template <size_t VolumeDim>
 class ElementId {
  public:
+  static constexpr size_t block_id_bits = 24;
+  static constexpr size_t grid_index_bits = 8;
+  static constexpr size_t refinement_bits = 4;
+  static constexpr size_t max_refinement_level = 16;
+  static constexpr size_t padding = 4;
```

Comment:
What is the purpose of this padding? Can't these bits just be absorbed by one of the other fields? Perhaps a code comment would be useful